packages/docs-ui/src/components/HeaderNavigation/HeaderNavigation.tsx

```diff
@@ -0,0 +1,43 @@
+import type React from 'react';
+import { Box, Hidden, HiddenVisually, Link, Text } from 'braid-design-system';
+import { MenuButton } from '../MenuButton/MenuButton';
+
+interface HeaderNavigationProps {
+  menuOpen?: boolean;
+  menuClick?: () => void;
+  logo: React.ReactNode;
```

Comment:
For consistency with the codebase, maybe just import the `ReactNode` type rather than `React`.